lib/cdo/rack/request.rb

```diff
@@ -1,5 +1,20 @@
+require 'rack/request'
+require 'ipaddr'
+
 module Rack
+  module CdoExtension
+    TRUSTED_PROXIES = JSON.parse(IO.read(deploy_dir('lib/cdo/trusted_proxies.json')))['ranges'].map do |proxy|
+      IPAddr.new(proxy)
+    end
+
+    def trusted_proxy?(ip)
+      super(ip) || TRUSTED_PROXIES.any?{|proxy| proxy === ip}
+    end
+  end
+
   class Request
+    prepend CdoExtension
```

Comment:
would it make sense to use `Rack::CdoExtension` in case there's some other random CdoExtension somewhere?